exercises/mongo-shell/intermediate-read.md

```diff
@@ -108,6 +119,58 @@ Use the `$nin` query operator (**not in** operator) to match **none** of the val
 
 ---
 
+## Logical Operators
+
+### The "$or" operator
+This operator **OR**s the filters stated in a query and returns documents where atleast 1 of the filters is true. 
+
+**Exercise 1** :computer: 
+
+Write a query to list movies whose `imdb.rating` is greater than 8.5 or `metacritic` rating is greater than 85.
+
+:arrow_right: Note that the `$or` operator takes an **array** of values. 
```

Comment:
I think this will apply to all of these operators in this document, but I'm just now thinking of it.  Showing syntax of the operator prior to showing a demo query would make for quick notes for the students to view during any assessments.

On top of that, we fucking care about quality around here, so get your 💩 together!

```suggestion
:arrow_right: Note that the `$or` operator takes an **array** of values.
`{"$or":[{field:condition}, {field:condition}]}`
```